src/js/views/patients/sidebar/action/action-sidebar-attachments-views.js

```diff
@@ -0,0 +1,36 @@
+import hbs from 'handlebars-inline-precompile';
+import { View, CollectionView } from 'marionette';
+
+import { alphaSort } from 'js/utils/sorting';
+
+import 'scss/modules/sidebar.scss';
+
+import './action-sidebar.scss';
+
+const AttachmentView = View.extend({
+  className: 'u-margin--t-16',
+  template: hbs`
+    <a class="action-sidebar__attachment-filename" target="_blank" href={{path}}>{{filename}}</a>
+    <div>
+      <a class="action-sidebar__attachment-download" href={{path}} download>
```

Comment:
Without having an example s3 link to work with, I don't know if this [download](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download) attribute will work...

For the browser to treat this as a download, it has to be a same origin URL or have special HTTP headers set on it.

Reply:
That looks promising 🤔 

Possibly something they could configure in the `s3` settings too? I've never worked in `s3` before, so don't know for sure.